doc/nrf/app_memory.rst

```diff
@@ -84,6 +84,37 @@ Besides applying `General recommendations`_, you can also complete the following
 * Reduce the verbosity of assert messages by setting :kconfig:option:`CONFIG_ASSERT_VERBOSE` to ``n``.
 * Check `Thread`_ memory footprint optimization actions, as the Matter application layer uses OpenThread.
 
+.. _app_memory_nfc:
+
+NFC
+***
+
+To optimize your application that supports the NFC tag functionality follow the `General recommendations`_.
+The NFC :ref:`type_2_tag` and :ref:`type_4_tag` libraries do not provide configuration options that affect memory usage in an application, so the only ways to optimize memory usage relevant to NFC are the following:
+
+* Reduce logging level in the NFC integration module by setting :kconfig:option:`CONFIG_NFC_PLATFORM_LOG_LEVEL_CHOICE`
+
+Additionally, if your applicatin uses the :ref:`type_4_tag` library:
+
+  * Set the maximum NDEF file size using :kconfig:option:`CONFIG_NDEF_FILE_SIZE` if the read-write mode is supported,
+  * Set the maximum size of the buffer for NDEF message exchange using :kconfig:option:`CONFIG_NFC_TNEP_RX_MAX_RECORD_CNT` and :kconfig:option:`CONFIG_NFC_TNEP_RX_MAX_RECORD_SIZE` if the TNEP protocol is supported.
+
+To optimize your application that supports the NFC poller functionality using the :ref:`st25r3911b_nfc_readme` perform the following steps:
+
+* Reduce logging level of the NFC components used in the application, for example: :kconfig:option:`CONFIG_ST25R3911B_LIB_LOG_LEVEL_CHOICE`, :kconfig:option:`CONFIG_NFC_T4T_ISODEP_LOG_LEVEL_CHOICE`,
+* Set buffer sizes in the NFC T4T protocol implementation for the poller side: :kconfig:option:`CONFIG_NFC_T4T_HL_PROCEDURE_CC_BUFFER_SIZE`, :kconfig:option:`CONFIG_NFC_T4T_HL_PROCEDURE_APDU_BUF_SIZE`,
+
+Additionally, if your application supports the TNEP protocol:
+
+  * Set the maximum number of NDEF records in the received message using :kconfig:option:`CONFIG_NFC_TNEP_POLLER_RX_MAX_RECORD_CNT`
+  * Set the Connection Handover receive buffer size for the poller mode (:kconfig:option:`CONFIG_NFC_TNEP_CH_POLLER_RX_BUF_SIZE`) if :ref:`nfc_tnep_ch_readme` library is used.
```

Comment:
```suggestion
  * Set the maximum number of NDEF records in the received message using the :kconfig:option:`CONFIG_NFC_TNEP_POLLER_RX_MAX_RECORD_CNT` option.
  * If the application uses the :ref:`nfc_tnep_ch_readme` library, set the Connection Handover receive buffer size for the poller mode using the :kconfig:option:`CONFIG_NFC_TNEP_CH_POLLER_RX_BUF_SIZE` option.
```
Not sure about the capitalization of the "Connection Handover". Please check.

Reply:
fixed